test/resource_test.rb

```diff
@@ -9,6 +9,8 @@ def setup
 
   def teardown
```

Comment:
Overall, I'm pretty happy with the test suite here! Good job. I honestly can't really think of other test cases to do... that's rare :)